src/tools/pub/cmd/index.md

```diff
@@ -2,29 +2,33 @@
 layout: default
 permalink: /tools/pub/cmd
 title: "Pub Commands"
-description: "Pub, a package and asset management tool for Dart, supports a variety of commands."
+description: "Pub, a package management tool for Dart, supports a variety of commands."
 ---
 
-The [pub tool](/tools/pub) has commands for managing packages,
-running and serving apps, and building and deploying web apps.
-
-Flutter has its own commands for managing and updating packages.
-For more information, see
-[Using Packages]({{site.flutter}}/using-packages/) and
-[Upgrading Flutter]({{site.flutter}}/upgrading/)
-on the [Flutter website.]({{site.flutter}})
+The [pub tool](/tools/pub) has commands for managing packages
+and for deploying packages and command-line apps.
+
+<aside class="alert alert-warning"><div class="alert-with-image">
+  <img src="{% asset_path flutter-logo.png %}" alt="[Flutter]">
+  <div markdown="1">
+  **Important:**
+  The Flutter SDK has its own commands for managing and updating packages.
+  For more information, see
+  [Using Packages]({{site.flutter}}/using-packages/) and
+  [Upgrading Flutter]({{site.flutter}}/upgrading/)
```

Comment:
I don't think you need the Upgrading Flutter link. The information how to upgrade is in the /using-packages link already.